test/scripts/README.md

```diff
@@ -0,0 +1,5 @@
+# Helper scripts
+
+This directory contains helper scripts used by Prow test jobs, as well and local development scripts.
+
+It lives in the `knative/test-infra` repository, and is exported to all other Knative repositories through the `git subtree split` branch `exported-scripts`, and usually lives in `test/scripts` in these repos.
```

Comment:
should vendor script go under serving/vendor/github.com/knative/test-infra/test/scripts?

Reply:
There's no go code there, thus `go dep` won't recognized it, and it will be removed or messed up every time you run `update-deps.sh` or `verify-codegen.sh`.